app/helpers/application_helper.rb

```diff
@@ -2,7 +2,7 @@
 
 module ApplicationHelper
   def link_to_switch_locale
-    if params[:locale] == 'en-US'
+    if I18n.locale == :'en-US'
```

Comment:
Can we switch language with this?I thought that we should look for `locale` in parameter to switch.

Reply:
@eitoball I made this PR based on my master so after merging another PR then we should be good: https://github.com/Safecast/safecastapi/pull/656